lib/modules/filteReddit.js

```diff
@@ -704,26 +904,41 @@ module.include = [
 	'search',
 ];
 module.exclude = [
-	// /^https?:\/\/(?:[-\w\.]+\.)?reddit\.com\/[-\w\.\/]+\/comments\/[-\w\.]+/i,
-	// /^https?:\/\/(?:[-\w\.]+\.)?reddit\.com\/comments\/[-\w\.]+/i
-	/^https?:\/\/(?:[\-\w\.]+\.)?reddit\.com\/over18.*/i,
+	/^\/over18.*/i,
 ];
 
-const excludeSaved = /^https?:\/\/(?:[\-\w\.]+\.)?reddit\.com\/user\/[\w]+\/saved/i;
-
 module.shouldRun = () => !(
 	module.options.excludeModqueue.value && isPageType('modqueue') ||
-	module.options.excludeUserPages.value && isPageType('profile')
+	module.options.excludeUserPages.value && isPageType('profile') ||
+	module.options.excludeCommentsPage.value && isPageType('comments')
 );
 
-module.beforeLoad = () => {
-	filterNSFW(module.options.NSFWfilter.value);
+let filterlineKey;
+let filterlineState;
+let initialFilterlineVisibility;
+let restorePage;
+
+module.beforeLoad = async () => {
+	updateNsfwBodyClass(module.options.NSFWfilter.value);
+
+	restorePage = browseContexts.currentLocation.getCurrent();
+
+	filterlineKey = `RESmodules.filteReddit.${restorePage.toLowerCase()}`;
+
+	filterlineState = await Storage.get(filterlineKey);
```

Comment:
Async storage ops in `beforeLoad()` aren't to be taken lightly since they block `go()` for all other modules.

Can we move the active status out of `filterlineState` and into a module option, so an extra storage fetch isn't necessary? (like [nightmodeOn](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/ac47977fe300c606757025bcbd2225539614ba29/lib/modules/nightMode.js#L24))

Then we can do the fetch just before `createFilterline` (maybe pass it down to where it's used so we don't need the global).